Sentiment analysis

ArthurPerry opened the pull request:
Please review the merge for sentiment-analysis demo

abbi-gaurav reviewed (changes requested):
- Please use `SAP Commerce Cloud` instead of `SAP Commerce`
- The steps to deploy are not there in the main README file. maybe they are yet to be added

ArthurPerry commented:
> * Please use `SAP Commerce Cloud` instead of `SAP Commerce`

I updated all references to `SAP Commerce Cloud`.  For the record `SAP Commerce` is a [valid product name](https://www.sapbrandtools.com/naming-center/#/requests/7751) and includes the on-premise product, for which this extension works exactly the same as the cloud version.  However as the main objective is for cloud solutions, I made the changes.

> * The steps to deploy are not there in the main README file. maybe they are yet to be added

I added deploy and verify steps to each README file.

ArthurPerry commented:
Full deployment steps are now included in the main README.md.

abbi-gaurav reviewed (approved):
I am approving the PR. A minor change required would be to use our standard format for prerequisites. (kyma runtime and kubernetes tooling)
https://github.com/SAP-samples/kyma-runtime-extension-samples/blob/main/address-completion-c4c/sales-cloud-address-completion-dqs/README.MD#prerequisites
